main.go

```diff
@@ -21,7 +21,8 @@ type Username string
 type config struct {
 	FlowdockAPIKey string `yaml:"flowdock_api_key"`
 	StoragePath    string `yaml:"storage_path"`
```

Comment:
Yes, will make it even more persistant. I started on a new refactoring branch so I'll include it there.